src/clj/game/cards/resources.clj

```diff
@@ -1417,6 +1417,30 @@
                  :delayed-completion true
                  :effect (effect (expose eid target) (trash card {:cause :ability-cost}))}]}
 
+   "Reclaim"
+   {:abilities
+    [{:label "[Click],[Trash],trash a card from your Grip: install a program, piece of hardware, or virtual resource from your Heap"
```

Comment:
I believe the click cost will already be appended to the label. Also, I think we only put the trash icon into the label if that is the only cost of the ability. (Based on code for Contract Killer, Alix, MCA Austerity.)